src/mk.sh

```diff
@@ -525,6 +525,14 @@ function mk_build()
   IFS=' ' read -r -a options <<< "$raw_options"
   for option in "${options[@]}"; do
     case "$option" in
+      --info|-i)
+        local kernel_version=$(get_kernel_release "$flag")
+        local compiled_modules=$(cat modules.order | wc -l)
```

Comment:
You are assuming that `modules.order` always exists, which is not true. This file is generated after users compile their kernel; in other words, do not `cat` directly to `modules.order`. You need to check if this file exists first, also, add a test to capture this case.

Reply:
Hmmmm... I was wondering, instead of relying on `cat modules.order | wc -l` (it require compilation), can we derive the total of drivers from being compiled from the `.config` file?

Reply:
If that's the case, what about doing  `grep -c =m .config`? to get the number of modules that will be compiled?

Reply:
Yes, I think this grep command probably works fine. Maybe compare `cat modules.order | wc -l` and `cat modules.order | wc -l` output.